docs/rest-api/source/includes/wp-api-v3/deposits.md

```diff
@@ -76,6 +76,9 @@ Fetch an overview of account deposits for all deposit currencies. This includes
         -   `weekly_anchor` _string_ | _undefined_ - The day of the week that payments are paid out, e.g. `monday`.
         -   `monthly_anchor` _int_ | _undefined_ - The day of the month that payments are paid out. Specified as a number between 1–31. 29-31 will instead use the last day of a shorter month.
     -   `default_currency` _string_ - The default currency for the account.
+	-   `default_external_accounts` _array_ - The default external accounts for the account.
```

Comment:
💭 can we offer more context here?

E.g. mention what kind of account this is (deposit destination, not Stripe account) and what `default` means (max one per currency).

Reply:
How about 
The default external payout acccounts (deposit destinations) for the Stripe account.

Reply:
💯 yeh nice. I added a small typo commit (acc~c~ounts) in 8b4cf9e.

Reply:
I hope you don't mind, I wrapped the example values in backticks e.g. `errored` in 519c448